afs/GettingStarted.md

```diff
@@ -0,0 +1,329 @@
+# Getting started with AFS
```

Comment:
Rename this file to README.md so that it will be automatically displayed in the GH browsing interface